lib/windows/launcher/models/appid.js

```diff
@@ -0,0 +1,3 @@
+import Immutable from 'immutable';
+
+export default Immutable.Record({ source: undefined, name: undefined });
```

Comment:
I find this model a bit unnecessary, could be part of the reducer and just pass `{ source, name }` in action to set it.

Reply:
That is how it works. This data type is in its own file because its used in views for prop validation.